### PR TITLE
[IMP] fieldservice: Support multiple equipments on service request

### PR DESCRIPTION
**Pull request by osi-scampbell**
This PR allows users to add multiple Equipment on a Service Request. It is important to note that only 1 Equipment can be used on a Repair Order or Maintenance Order. So when those are selected, we hide the Many2many field and make the user select 1 Equipment.

### CODE REVIEW

File: fieldservice/views/fsm_order.xml [hunk 1]
```diff
@@ -158,6 +159,20 @@
                                 <group id="execution-right"/>
                             </group>
                         </page>
+                        <page string="Equipment"
```
Comment (max3903) [inline fieldservice/views/fsm_order.xml]:
```suggestion
                        <page string="Equipments" groups="fieldservice.group_fsm_equipment"
```

File: fieldservice/views/fsm_order.xml [hunk 2]
```diff
@@ -158,6 +159,20 @@
...
+                            attrs="{'invisible': [('type', 'in', ['repair', 'maintenance'])]}">
+                            <field name="equipment_ids"
+                                   attrs="{'required': [('type', 'not in', ['repair', 'maintenance'])]}"
+                                   groups="fieldservice.group_fsm_equipment"
```
Comment (max3903) [inline fieldservice/views/fsm_order.xml]:
```suggestion
```

**Comment by max3903** [comment]
@barnarj and @kvedder-amplex If you get a chance, can you review/test this branch and provide your feedbacks?

**Review by max3903 — APPROVED** [review]

File: fieldservice_repair/models/fsm_order.py [hunk 3]
```diff
@@ -12,32 +12,32 @@ class FSMOrder(models.Model):
...
-                    'partner_id':
-                        order.customer_id and order.customer_id.id or False,
-                })
-                order.repair_id = repair_id
-            elif not order.equipment_id.current_stock_location_id:
-                raise ValidationError(_("Cannot create Repair Order because "
-                                        "Equipment does not have a Current "
-                                        "Inventory Location."))
-        return order
+    # @api.model
```
Comment (marcelsavegnago) [inline fieldservice_repair/models/fsm_order.py:15]: Hi @osi-scampbell, I tested the fieldservice_repair module, but I saw that the CREATE method was commented. Do you have more information on why? Do you see any problem with uncommenting this method?

**Pull request merged**